docs/best-practices/mocking-request.md

```diff
@@ -5,17 +5,20 @@ Undici have its own mocking [utility](../api/MockAgent.md). It allow us to inter
 Example:
 
 ```js
-// index.mjs
+// bank.mjs
```

Comment:
Renamed to `bank.mjs`. The `import` in the example below was incorrect by referencing `undici.mjs`. Happy to change this comment to `undici.mjs` if preferred.